src/components/suggestions.ts

```diff
@@ -57,6 +62,21 @@ export const addSuggestion = async (
   );
 };
 
+export const updateSuggestionCron = async (ids: number[], state = 'pending'): Promise<void> => {
```

Comment:
should rename to `updateSuggstionsState`, having cron in the name couples this function with the cron component, which is not what we want